### ISSUE TITLE
✨ "Restore window frame on drag" should be for mouse commands only

### ISSUE THREAD

**Issue by Stooovie** [MrKai77/Loop#320]
### Feature Request Description

When "Restore window frame on drag" is enabled, any and all windows dragging is considered as a restore. 

**That results in an inability to move windows without them restoring the original size.**

In my opinion, "Restore window frame on drag" should only ever happen when the window was snapped by dragging the window with the mouse in the first place. If the user used keyboard instead to get into a specific position and scale, dragging to restore should not be happening.

### Final Checks

- [X] My issue title is descriptive
- [X] This is a single feature request (multiple feature requests should be reported individually)

**Comment by MrKai77** [comment]
Yes, I definitely agree! I will try and implement this in the near future!

**Comment by soul-ride** [comment]
Hi,

It seems like it's still working as is. That issue actually forced me to disable "Restore window frame on drag" feature. Are there any plans for a fix?

Thanks.
